src/nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -213,8 +216,11 @@ def run_conversion(
             if external_mode:
                 with VideoCaptureContext(str(file_list[0])) as vc:
                     fps = vc.get_movie_fps()
+                    if timestamps is None:
+                        timestamps = starting_times[j] + vc.get_movie_timestamps()
```

Comment:
If passing `timestamps`, `starting_times` should not be passed (one-or-the-other kind of thing). The `timestamps` values should be incremented to the `session_start_time` before passing into here.